core/src/main/java/Scenes/BaseScreen.java

```diff
@@ -22,23 +28,44 @@
   protected final OrthographicCamera camera;
   protected final FilePlayMain mainGame;
   protected final Stage stage;
+  protected TextureAtlas buttonAtlas;
+  protected final Sprite banner;
+  protected GlyphLayout bannerTextGlyphLayout;
+  protected BitmapFont bannerText;
+  protected Skin buttonSkin;
   private FreeTypeFontGenerator generator;
   private FreeTypeFontParameter parameter;
 
   // This is the native screen size that will be the reference for everything placed on the screen.
   protected static final float WORLD_WIDTH = 480;
   protected static final float WORLD_HEIGHT = 800;
 
+  // Used to calculate where to draw the text in the banner so that it is centered.
+  protected int glyphCenterX;
+
   // Ratio of world units and pixels of a screen.
   protected float HeightWorldPixelRatio = WORLD_HEIGHT / (float) Gdx.graphics.getHeight();
   protected float WidthWorldPixelRatio = WORLD_WIDTH / Gdx.graphics.getWidth();
 
   BaseScreen(FilePlayMain mainGame) {
     this.mainGame = mainGame;
     spriteBatch = new SpriteBatch();
+    // Creates new FreeTypeFontParameter to modify fonts.
+    parameter = new FreeTypeFontParameter();
+    // Creates BitmapFont for the text that is going to be in the center of the banner.
+    bannerText = new BitmapFont();
+    bannerText = generateNewFont("BROADSolid.ttf", 36, Color.BLACK);

```

Comment:
You're creating a new BitmapFont when you say "bannerText = new BitmapFont()", but then you're replacing that newly instantiated BitmapFont with the object returned by the call to "generateNewFont()".